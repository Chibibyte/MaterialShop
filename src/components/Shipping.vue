<template>
  <ContentBase
    :validate="validate"
    :dense="dense"
    :value="value"
    :disabled="disabled"
    :readonly="readonly"
    :contentMeta="contentMeta"
    @input="v => $emit('input', v)"
  />
</template>

<script>
import ContentBase from "@/components/ContentBase.vue";
let cb = ContentBase.methods.createMe(
  ["name", "Delivery Service"],
  ["price", "Price"]
);
cb.components = {
  ContentBase
};
cb.name = "Shipping";
export default cb;
</script>