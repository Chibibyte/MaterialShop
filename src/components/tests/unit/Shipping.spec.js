import { shallowMount } from '@vue/test-utils';
import Shipping from '@/components/Shipping.vue';

let stubs = [
]

let wrapper;
beforeEach(() => {
    wrapper = shallowMount(Shipping, { stubs });
})

describe('Component', () => {
    test('is instantiated', () => {
        expect(wrapper.isVueInstance).toBeTruthy();
    })

    test('default snapshot', () => {
        expect(wrapper.element).toMatchSnapshot();
    })
})