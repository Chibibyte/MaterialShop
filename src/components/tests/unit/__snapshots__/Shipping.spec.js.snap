// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component default snapshot 1`] = `
<contentbase-stub
  contentmeta="[object Object],[object Object]"
  validate="default() {}"
  value="[object Object]"
/>
`;
